python/hail/expr/functions.py

```diff
@@ -2626,7 +2660,7 @@ def abs(x):
 
 
 @typecheck(x=oneof(expr_numeric, expr_array(expr_numeric)))
-def signum(x):
+def signum(x: Num_T) -> Num_T:
```

Comment:
I would have thought this always returns `Int32`.

Reply:
you're totally right, judging from the code.